vechile_info :
  Motor:
    avg_starter_Crnt :
      avg_starter_Crnt_min :
      avg_starter_Crnt_max :
    avgMtr_phaseV :
      avgMtr_phaseV_min :
      avgMtr_phaseV_max:
    tgt_torq :
      tgt_torq_min :
      tgt_torq_max :
    mtractl_torq :
      mtractl_torq_min :
      mtractl_torq_max :
    propspeedlmt :
      propspeedlmt_min :
      propspeedlmt_max :
    MotorRPM :
      MotorRPM_min :
      MotorRPM_max :
    calbattery_crnt :
      calbattery_crnt_min :
      calbattery_crnt_max :
    ctrlcapctr_v :
      ctrlcapctr_v_min :
      ctrlcapctr_v_max :
    trottleip :
      trottleip_min :
      trottleip_max :
    mtr_temp :
      mtr_temp_min :
      mtr_temp_max :
    ctrlr_tmrp :
      ctrlr_tmrp_min :
      ctrlr_tmrp_max :
    dis_traveld :
      dis_traveld :
      dis_traveld :
  Battery:
    avg_starter_Crntp :
      avg_starter_Crntp_min :
      avg_starter_Crntp_max :
    batterycurrent :
      batterycurrent_min :
      batterycurrent_max :
    drive_dirctn :
      drive_dirctn_min :
      drive_dirctn_max :
    batteryvoltage :
      batteryvoltage_min :
      batteryvoltage_max :
    batterysoc :
      batterysoc_min :
      batterysoc_max :
    batteryamphours :
      batteryamphours_min :
      batteryamphours_max :
    batteryhighesttemperature :
      batteryhighesttemperature_min :
      batteryhighesttemperature_max :
    batterylowesttemperature :
      batterylowesttemperature_min :
      batterylowesttemperature_max :
    batteryaveragetemperature :
      batteryaveragetemperature_min :
      batteryaveragetemperature_max :
  steering :
    torque :
      torque_min :
      torque_max :
    motor_duty :
      motor_duty_min :
      motor_duty_max :
    current :
      current_min :
      current_max :
    supplyvoltage :
      supplyvoltage_min :
      supplyvoltage_max :
    switch_position :
      switch_position_min :
      switch_position_max :
    box_temperature :
      box_temperature_min :
      box_temperature_max :
    torqueA :
      torqueA_min :
      torqueA_max :
    torqueB :
      torqueB_min :
      torqueB_max :
